tools/cellprofiler/macros.xml

```diff
@@ -44,6 +48,10 @@
         ]]></command>
     </xml>
 
+    <xml name="text_validator">
+        <validator type="regex" message="Only numbers, letters, hyphen and underscore and spaces are allowed">^[A-Za-z0-9 _-]*$</validator>
```

Comment:
is space actually working? can we add a test for this?

Reply:
yes it works.. there's a space after "9", I have tested.
the test cases in module "measure_image_quality" has parameters which contains spaces as value .